TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I agree we don't need the absolute minimum for the first checkin. But for things that can obviously be removed, it is best to remove them now, or else they will stay around for a long time.